aiida/engine/processes/functions.py

```diff
@@ -371,7 +372,8 @@ def build(func: FunctionType, node_class: t.Type['ProcessNode']) -> t.Type['Func
         signature = inspect.signature(func)  # type: ignore[unreachable]
 
         args: list[str] = []
-        varargs: str | None = None
+        var_positional: str | None = None
+        var_keyword: str | None = None
         keywords: str | None = None
```

Comment:
Since you added a new `var_keyword`. Please remove the old `keyword`. also in the code after.
```suggestion
        var_keyword: str | None = None
```

Reply:
Done